tests/speedup.scm

```diff
@@ -53,6 +53,12 @@
         (lp (1+ i) (make-vector (- words 2) #f))
         x)))
 
+(unless (getenv "FIBERS_EXPENSIVE_TESTS")
```

Comment:
https://github.com/wingo/fibers/issues/104 is relevant.

I consider this name misleading. Proposal: "FIBERS_SKIP_BENCHMARKS".

Reply:
Addressed in #113 (we can still change the name of the variable if anyone feels strongly about it).